Enhance sign-out guidance (#37198)
dotnet/AspNetCore.Docs: aspnetcore/blazor/security/index.md

> [!NOTE]
> Each browser tab requires a separate circuit, so additional tabs opened by a user don't observe an authentication state change until their circuits revalidate. However, the approach in this section sets the revalidation interval for all of the tabs opened by a user.

> To control the revalidation interval of the underlying authentication cookie in apps that adopt ASP.NET Core Identity, see the following [Sign out for ASP.NET Core Identity](#sign-out-for-aspnet-core-identity) subsection for details. For apps that adopt cookie-based authentication without Identity, see the following [Sign out for cookie-based authentication](#sign-out-for-cookie-based-authentication) subsection.

Reviewer: Drop "the underlying authentication cookie" (the Identity subsection is about the security-stamp validator, not the cookie itself) and clarify that this applies to Identity with cookie authentication — the revalidation interval doesn't apply to Identity bearer tokens, which rely on the refresh interval instead.

```suggestion
To control the revalidation interval in apps that adopt ASP.NET Core Identity with cookie authentication, see the following [Sign out for ASP.NET Core Identity](#sign-out-for-aspnet-core-identity) subsection for details. For apps that adopt cookie-based authentication without Identity, see the following [Sign out for cookie-based authentication](#sign-out-for-cookie-based-authentication) subsection.
```

The reference source for <xref:Microsoft.AspNetCore.Components.Server.RevalidatingServerAuthenticationStateProvider> is located in [`RevalidatingServerAuthenticationStateProvider.cs`](https://github.com/dotnet/aspnetcore/blob/main/src/Components/Server/src/Circuits/RevalidatingServerAuthenticationStateProvider.cs).

> [!INCLUDE[](~/includes/aspnetcore-repo-ref-source-links.md)]

Reviewer: This duplicates the reference-source link from line 539 and re-includes the same `aspnetcore-repo-ref-source-links.md` footer note that's already included at line 541, which will render the standard note twice on the page. Suggest removing both.

```suggestion
```

Author: It's in a different section. However ... yes ... let's delete on the grounds it was already mentioned and make the article shorter.

#### Sign out for ASP.NET Core Identity

> To proactively, completely sign a user off within less than the default 30 minute revalidation interval in apps that adopt ASP.NET Core Identity, use the guidance in this section.

Reviewer: "To proactively, completely sign a user off" reads awkwardly with the stacked adverbs.

```suggestion
To force a complete sign-out within less than the default 30-minute revalidation interval in apps that adopt ASP.NET Core Identity, use the guidance in this section.
```
